test/integration/testUtil.js

```diff
@@ -101,6 +101,23 @@ module.exports.dropTablesIgnoringErrorsAsync = async (connection, tableNames) =>
   }
 };
 
+/**
+ * Drop databases one by one if exist - any connection error is ignored
+ * @param connection Connection
+ * @param dbNames string[]
+ * @return {Promise<void>}
+ */
+module.exports.dropDBsIgnoringErrorsAsync = async (connection, dbNames) => {
+  for (const dbIdx in dbNames) {
```

Comment:
let's prefer `dbNames.forEach` and don't use index when unnecessary

Reply:
My codes are async and it seems like async is not working with forEach. https://gist.github.com/joeytwiddle/37d2085425c049629b80956d3c618971. Instead of  the foreach, I used map function isntead.